ddtrace/span.py

```diff
@@ -528,6 +533,40 @@ def context(self):
             self._context = Context(trace_id=self.trace_id, span_id=self.span_id)
         return self._context
 
+    def link_span(self, context, attributes=None):
+        # type: (Context, Optional[Dict[str, Any]]) -> None
+        """Defines a causal relationship between two spans"""
+        if not context.trace_id or not context.span_id:
+            raise ValueError("Invalid span or trace id. trace_id:{context.trace_id} span_id:{context.span_id}")
```

Comment:
`.format` missing?

Reply:
I wanted to make this an f string. I will fix this